br/pkg/storage/gcs.go

```diff
@@ -270,11 +276,8 @@ func (s *GCSStorage) URI() string {
 
 // Create implements ExternalStorage interface.
 func (s *GCSStorage) Create(ctx context.Context, name string, _ *WriterOption) (ExternalFileWriter, error) {
```

Comment:
There's field `Concurrency` in `WriterOption`, we can use it to limit the error group concurrency.